overlord/snapstate/snapstate.go

```diff
@@ -2469,7 +2470,7 @@ var RestoreValidationSetsTracking func(st *state.State) error
 // AutoRefresh is the wrapper that will do a refresh of all the installed
 // snaps on the system. In addition to that it will also refresh important
 // assertions.
-func AutoRefresh(ctx context.Context, st *state.State) ([]string, *UpdateTaskSets, error) {
+func AutoRefresh(ctx context.Context, st *state.State, isContinuedAutoRefresh bool) ([]string, *UpdateTaskSets, error) {
```

Comment:
An alternative would be to take `Flags` here so we're automatically documenting exactly what parameter is `true`. 
`AutoRefresh(..., true)`
vs
`AutoRefresh(..., &Flags{IsContinuedAutoRefresh: true})`

It's also more easily extended if we ever need to add other parameters

Reply:
I added a new AutoRefreshOptions now as Flags might be a bit too broad.